tools/lsp/common.rs

```diff
@@ -46,40 +42,17 @@ pub struct PreviewComponent {
     /// If None, then the last component is going to be shown.
     pub component: Option<String>,
 
-    /// The list of include paths
-    pub include_paths: Vec<PathBuf>,
-
-    /// The map of library paths
-    pub library_paths: HashMap<String, PathBuf>,
-
     /// The style name for the preview
     pub style: String,
 }
 
 #[allow(unused)]
 #[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
 pub enum LspToPreviewMessage {
-    SetContents {
-        path: String,
-        contents: String,
-    },
-    SetConfiguration {
-        show_preview_ui: bool,
-        style: String,
-        include_paths: Vec<String>,
-        library_paths: Vec<(String, String)>,
-    },
-    ShowPreview {
-        path: String,
-        component: Option<String>,
-        style: String,
-        include_paths: Vec<String>,
-        library_paths: Vec<(String, String)>,
-    },
-    HighlightFromEditor {
-        path: Option<String>,
-        offset: u32,
-    },
+    SetContents { path: String, contents: String },
+    SetConfiguration { config: PreviewConfig },
```

Comment:
This seems to "work" fine.  
both with native preview, wasm preview, and web extension.
Setting the library or include path is properly transfered to the preview.  
(but of course it doesn't work with wasm as it can't open the path, which is another issue)

Could you elaborate what broke and why the mapping is necessary?
Perhaps it was just a temporary bug in a dependency?